Tests/src/test/scala/equellatests/pages/LoginNoticePage.scala

```diff
@@ -0,0 +1,57 @@
+package equellatests.pages
+
+import com.tle.webtests.framework.PageContext
+import equellatests.browserpage.NewTitledPage
+import org.openqa.selenium.{Keys, WebElement}
+
+case class LoginNoticePage(ctx: PageContext) extends NewTitledPage("Login Notice Editor", "page/loginconfiguration") {
+
+  def preNoticeApplyButton:WebElement = findElementById("preApplyButton")
+
+  def preNoticeDeleteButton:WebElement = findElementById("preDeleteButton")
+
+  def preNoticeField:WebElement = findElementById("preNoticeField")
+
+  def postNoticeApplyButton:WebElement = findElementById("postApplyButton")
+
+  def postNoticeDeleteButton:WebElement = findElementById("postDeleteButton")
+
+  def postNoticeField:WebElement = findElementById("postNoticeField")
+  
+  def clickPreApply(): Unit = {
+    this.preNoticeApplyButton.click()
+  }
+
+  def clickPreDelete(): Unit = {
+    this.preNoticeDeleteButton.click()
+  }
+
+  def populatePreNoticeField(notice:String): Unit = {
+
+    preNoticeField.sendKeys(Keys.chord(Keys.CONTROL, "a"))
+    preNoticeField.sendKeys(Keys.DELETE)
+    preNoticeField.sendKeys(notice)
```

Comment:
This is a bit interesting, as typically doing the sendKeys is slower than just setting the fields value. You normally only do the sendKeys if you really have to.

HOWEVER, is this the oEQ autotests idiom? If so, well.....